MujocoPlugins:
  - type: mujoco_ros_mocap/MocapPlugin
  - type: mujoco_contact_surfaces/MujocoContactSurfacesPlugin
    SurfacePlugins:   
    - {
      type: mujoco_contact_surface_sensors/CurvedSensor,
      sensorName: "myrmex_fingertip",
      geomName: "fingertip_geom",
      topicName: "/myrmex_fingertip",
      updateRate: 4.0,
      visualize: True,
      visualize_max_pressure: 0.000000003,
      include_margin: 0.006,
      method: "squared",
      sample_method: "area_importance",
      sample_resolution: 0.0005,
      taxels:
      [
        [ 0.00415178164, -0.00615073064, 0.01464621212],
        [ 0.008619488, 0.00046286059, 0.01316570371],
        [ 0.00830223182, 0.00074051459, 0.01945107626],
        [ 0.00354169091, -0.00500616896, 0.0238568657],
        [ 0.00266992694, 0.00055709812, 0.03028623604],
        [ 0.00713842137, 0.00097836545, 0.02588591432],
        [ -0.00713842136, 0.00097836546, 0.02588591435],
        [ -0.00266992693, 0.00055709811, 0.030286236 ],
        [ -0.00354169087, -0.00500616897, 0.02385686587],
        [ -0.00830223175, 0.00074051458, 0.01945107617],
        [ -0.00861948819, 0.00046286058, 0.01316570354],
        [ -0.0041517816, -0.0061507306, 0.01464621205],        
      ],
      normals:
      [
        [ 0.37386554, -0.92331819, 0.08779566],
        [0.9989708, -0.03265917, 0.03147561],
        [ 0.99543803, -0.03717611, 0.08786959],
        [0.36603357, -0.89345384, 0.26030685],
        [0.35214134, -0.13784856, 0.92573984],
        [0.9230213, -0.06311386, 0.37953699],
        [-0.9230213, -0.06311386, 0.37953699],        
        [-0.35214134, -0.13784856, 0.92573984],
        [-0.36603357, -0.89345384, 0.26030685],
        [-0.99543803, -0.03717611, 0.08786959],
        [-0.9989708, -0.03265917, 0.03147561],
        [-0.37386554, -0.92331819, 0.08779566],
      ],
    }
      